### DISCUSSION TITLE
SixLabors.ImageSharp update from 1.0.0-beta0009 to 2.0.0

### DISCUSSION THREAD

**Discussion by angelojjj** [SixLabors/ImageSharp#2642] [Q&A]
I am currently using SixLabors.ImageSharp 1.0.0-beta0009 in my .NET Core app. The startup.cs has the following:

        services.AddImageSharp()
            .SetMemoryAllocator(provider => ArrayPoolMemoryAllocator.CreateWithMinimalPooling());
I updated is to 2.0.0 and now I get an error saying "SetMemoryAllocator" is not available.

Is it alright to just get rid of SetMemoryAllocator? and just do services.AddImageSharp()

From SixLabors website and I found the folling:

Starting with ImageSharp 2.0, the library uses large (~4MB) discontiguous chunks of unmanaged memory to represent multi-megapixel images. Internally, these buffers are heavily pooled to reduce OS allocation overhead.

Which makes me think that's why they do not have SetMemoryAllocator anymore. Am I right?

**Comment by JimBobSquarePants** [comment]
You didn’t need to set the allocator anyway. The pooled one was the default for V1.

However, you should aim for targeting the latest V3 version.

**Comment by JimBobSquarePants** [comment]
Yeah, you don’t need this. Also try and use a more recent version if you can.
